### ISSUE TITLE
Refactor APageFactory, rename it PageFactoryImpl

### ISSUE THREAD

**Issue by stevesobol** [cjwizard/cjwizard#70]
`APageFactory` implements the `PageFactory` interface, but the class name doesn't reflect that fact.

**Comment by spyhunter99** [comment]
this would probably be a breaking change, recommend 1.1.x version change or 2.0.0

**Comment by spyhunter99** [comment]
i can do this once the demo pr is merged

**Comment by stevesobol** [comment]
This isn't a necessity. I just think it's a good idea. It's relatively low priority; let's take care of the other pull requests first.

**Comment by stevesobol** [comment]
Merged [your pull request](https://github.com/cjwizard/cjwizard/pull/80).
